<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadsWidget</class>
 <widget class="QWidget" name="DownloadsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QHeaderView::section {
    background-color: #f9f9f9;
    color: rgb(27, 105, 165);
    border: none;
    border-left: 1px solid #f0f0f0;
    font-weight: normal;
    font-size: 12pt;
    padding-top: 4px;
    padding-bottom: 4px;
    margin-top: 1px;
    margin-bottom: 1px;
}

QHeaderView::section:first {
    border: none;
}

QHeaderView::section:checked
{
}

#tableImports, #tablePackages {
	alternate-background-color: rgb(243, 246, 250);
    border: none;
	font-size: 11px;
	border-top: 3px solid;
	border-bottom: 3px solid;
	border-top-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(100, 100, 100, 255), stop:0.2 rgba(100, 100, 100, 50), stop:1 rgba(255, 255, 255, 0));
	border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(100, 100, 100, 255), stop:0.2 rgba(100, 100, 100, 50), stop:1 rgba(255, 255, 255, 0));
	selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #4185b6, stop:1 #1b6aa5);
	color: #395265;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelArchives">
     <property name="font">
      <font>
       <family>Pathway Gothic One</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #224f7f;
</string>
     </property>
     <property name="text">
      <string>Archives</string>
     </property>
     <property name="indent">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tablePackages">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Files</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelImportable">
     <property name="font">
      <font>
       <family>Pathway Gothic One</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #224f7f;
</string>
     </property>
     <property name="text">
      <string>Importable items</string>
     </property>
     <property name="indent">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableImports">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>170</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Files</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
